docs/4.3/enterprise/quickstart-enterprise.md

```diff
@@ -315,6 +320,101 @@ Any SAML-compliant provider can be configured with Teleport by following the
 same steps.  There are Teleport Enterprise customers who are using Oracle IDM,
 SailPoint and others.
 
+## Run Teleport Enterprise using Docker
+
+We provide pre-built Docker images for every version of Teleport Enterprise. These images are hosted on quay.io.
+
+- [All tags under `quay.io/gravitational/teleport-ent` are Teleport Enterprise images](https://quay.io/repository/gravitational/teleport-ent?tag=latest&tab=tags)
+
+We currently only offer Docker images for `x86_64` architectures.
+
+### Pick your image
+
+This table gives an idea of how our image naming scheme works. We offer images which point to a static version of Teleport Enterprise, as well as images which are
+automatically rebuilt every night. These nightly images point to the latest version of Teleport Enterprise from the current release branch, plus the two previous release branches.
```

Comment:
nit: how about "from the three most recent release branches"?

Reply:
Much better.